README.md

```diff
@@ -85,7 +85,7 @@ $ brew install carthage
 To integrate Alamofire into your Xcode project using Carthage, specify it in your `Cartfile`:
 
 ```ogdl
-github "Alamofire/Alamofire" "swift-2.0"
+github "Alamofire/Alamofire" ~> 2.0
```

Comment:
`~> 2.0.0` and `~> 2.0` mean different things. `~> 2.0` means that any minor version of 2.0 is allowed whereas `~> 2.0.0` means only patch level changes (minor and major must be 2.0).